### ISSUE TITLE
LOL  requesting more information.

### ISSUE THREAD

**Issue by Racing4funn** [SpektreWorks/spektreworks.github.io#7]
I saw the manual page on your site but it seems to be more so an over view of what would be unique to your board and a collections of Watch out done make this mistake.  Is there a detailed manual or schematics?

Maybe this board is marketed towards advance builders but this one is my first and well, not advance here.

If there is a forum I can find most my answers please provide a link.. Thanks!

**Comment by SpektreWorks** [comment]
Indeed, our product is aimed towards users that are already familiar with the platform. If you're new, there's a wealth of information and guides online. The best place to start is the official ArduPilot documentation at http://ardupilot.org/ardupilot/ . There is also a forum community on the ardupilot.org website.

If you have any specific questions about our carrier board, feel free to ask them here.
